wallet/src/main/java/com/android/identity_credential/wallet/logging/Event.kt

```diff
@@ -0,0 +1,24 @@
+package com.android.identity_credential.wallet.logging
+
+import com.android.identity.cbor.Cbor
+import com.android.identity.cbor.annotation.CborSerializable
+import com.android.identity.util.Logger
+import kotlinx.datetime.Instant
+
+@CborSerializable
+sealed class Event(
+    open val timestamp: Instant,
+    open var eventId: Long = EventLogger.generateUniqueID(),
+    open var documentId: String = "",
```

Comment:
So far the two events that are in scope have a documentID, but if you know that other events won't need it, then yeah it makes sense to push it down to subclasses.

Reply:
We will absolutely have events without documentID in the future, for example when refreshing trust lists from a VICAL (which I'm working on right now). Since we are persisting things on disk, getting the data model right the first time is kinda important.

Reply:
Done.